src/pkg/message/message.go

```diff
@@ -215,6 +215,7 @@ func Notef(format string, a ...any) {
 	notePrefix.Println(message)
 }
 
+// Title prints a title and an optional help description for that section
 func Title(title string, help string) {
```

Comment:
like `NOTE` and `WARNING` I think this would do well to also have a space padding before and after the title

Reply:
For my two-cents, I think I like the way it looks without space padding but with it being formatted as a header.

Reply:
Adding my thoughts here from slack thread. I think we should not have a blank line between line and section header. Do not agree with using large grey headers as they may interrupt scalability of the file as a whole. 

As is feels like sections. 
Big headers may cause a user not to look above what ever section that is visible

Reply:
Two changes after discussion:

1. Add a space before and after title like ` WARNING ` on titles (i.e. ` Potential Breaking Changes `)
2. Make `HeaderInfof` print a 100 column wide header instead of 85

Reply:
This now looks like this:
![image](https://github.com/defenseunicorns/zarf/assets/3977569/82315af8-c043-4eeb-b15e-6339f69c324a)